src/main/resources/edu/cornell/kfs/module/purap/cu-ojb-purap.xml

```diff
@@ -3495,4 +3495,14 @@
     <field-descriptor name="cmForeignCurrencyTypeCode" column="CM_FCRNCY_TYP_CD" jdbc-type="VARCHAR" />
 </class-descriptor>
 
+	<class-descriptor class="edu.cornell.kfs.module.purap.businessobject.JaggaerRoleLinkMapping" table="CU_JAGGAER_ROLE_LINK_T">
```

Comment:
This entry is missing a field descriptor for "versionNumber".

Reply:
oh, good catch, thanks